Makefile

```diff
@@ -164,6 +164,11 @@ image-%: ## Builds the specified image. Valid options are aws, azure, digital-oc
 
 images: image-aws image-azure image-digital-ocean image-gcp image-metal image-openstack image-vmware ## Builds all known images (AWS, Azure, Digital Ocean, GCP, Metal, Openstack, and VMware).
 
+sbc-%: ## Builds the specified SBC image. Valid options are rpi_4, bananapi_m64, and libretech_all_h3_cc_h5 (e.g. sbc-rpi_4)
```

Comment:
We need to either pass `console` kernel args here or allow boards to add extra kernel args at install time (like platforms do)

Reply:
I vote for the platform-like interface, as this should allow us to install from PXE as well? if we put `board:` to the install section?